code/game/objects/effects/effect_system.dm

```diff
@@ -69,7 +69,8 @@ steam.start() -- spawns the effect
 /datum/effect/effect/system/steam_spread/start()
 	var/i = 0
 	for(i=0, i<src.number, i++)
-		addtimer(CALLBACK(src, /datum/effect/effect/system/proc/spread, i), 0)
+		spread(i)
+		//addtimer(CALLBACK(src, /datum/effect/effect/system/proc/spread, i), 0)
```

Comment:
Why was this changed? I believe the 0 delay callback is intentional.

Reply:
It was removed because it caused massive lag when you had a few geysers on the map, and a lot of timer runtimes. Plus, spread already is async anyways?
I actually rewrote that effect for geysers using particles though.

Reply:
I'll revert that